CONTRIBUTING.rst

```diff
@@ -73,6 +70,11 @@ Now you are ready to setup your development environment using the following comm
 This will create a new virtualenv and install the dependencies for using ics.py.
 Furthermore, the current source of the ics.py package will be available similar to running ``./setup.py develop``.
 To access the virtualenv, use ``poetry run python`` or ``poetry shell``.
+The :file:`poetry.lock` file locks the versions of dependencies in the development environment set up by poetry, so that this environment is the same for everyone.
+The file is only read by poetry and not included in any distributions, so these restrictions don't apply when running ``pip install ics``.
+As tox manages its own environments and also doesn't read the lock file, it just installs the latest versions of dependencies for testing.
+More details on the poetry side can be found `here <https://python-poetry.org/docs/basic-usage/#commit-your-poetrylock-file-to-version-control>`_.
+
```

Comment:
I'd have a small suggestion. :wink: I would shorten some sentences, remove weasel words etc. so here is the slightly changed version:

```suggestion
The :file:`poetry.lock` file locks the versions of dependencies in the development environment set up by poetry. This ensures that such an environment is the same for everyone.
The file :file:`poetry.lock` is only read by poetry and not included in any distributions. These restrictions don't apply when running ``pip install ics``.
As tox manages its own environments and doesn't read the lock file, it installs the latest versions of dependencies for testing.
More details on the poetry side can be found in the `poetry documentation <https://python-poetry.org/docs/basic-usage/#commit-your-poetrylock-file-to-version-control>`_.

```